easybuild/easyconfigs/g/GBprocesS/GBprocesS-2.1.1-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonPackage'
+
+name = 'GBprocesS'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gbprocess.readthedocs.io/'
+description = """GBprocesS allows for the extraction of genomic inserts from NGS
+data for GBS experiments. Preprocessing is performed in different stages that
+are part of a linear pipeline where the steps are performed in order. GBprocesS
+provides a flexible way to adjust the functionality to your needs, as the
+operations required and the execution order vary depending on the GBS protocol
+used."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = ['https://gitlab.com/dschaumont/GBprocesS/-/archive/%(version)s/']
+sources = ['GBprocesS-%(version)s.tar.gz']
+checksums = ['968e9431160d4c552e85e6478d79e22f3c1d946130917b3cee0a6c602d51e5e1']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('cutadapt', '2.10', '-Python-%(pyver)s'),
+    ('Biopython', '1.78', versionsuffix),
+    ('fastq-join', '1.3.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
+
+# Tests expect the package to already be installed, but EB runs them beforehand
+pretestopts = (
+    "sed -i 's/"
+    "__version__ = get_distribution(__name__).version/"
+    "__version__ = \"%(version)s\"/g' gbprocess/__init__.py && "
+    "sed -i \"s|use_scm_version={'write_to': 'gbprocess/version.py'}|version='%(version)s'|g\" setup.py && "
```

Comment:
Maybe we can avoid this by running the tests through `sanity_check_commands`?

```python
sanity_check_commands = ["cd %(builddir)s && python -m unittest discover -v -s ./test -p test_*.py"]

Reply:
It doesn't seem that way, not without using git_config instead of a tarball - setuptools-scm can't provide `get_distribution(__name__).version` without a .git/ directory. So I think this is the better option :man_shrugging: